wazo_confgend/confgen.py

```diff
@@ -21,13 +19,37 @@
 from wazo_confgend.template import new_template_helper
 from xivo_dao.helpers.db_utils import session_scope
 from twisted.internet.protocol import Protocol, ServerFactory
+import twisted.python.failure
 
 logger = logging.getLogger(__name__)
 
 
 class Confgen(Protocol):
+    def connectionMade(self):
+        logger.info("connection established")
+
+    def connectionLost(self, reason: twisted.python.failure.Failure):
+        logger.info("connection lost: %s", reason.getErrorMessage())
+        super().connectionLost(reason=reason)
 
-    def dataReceived(self, data):
+    def commandReceived(self, cmd: str, args: List[str]):
+        logger.debug(
+            "command received cmd_length=%d, args_count=%d", len(cmd), len(args)
+        )
+        try:
+            resource, filename = cmd.split('/')
```

Comment:
Does this always have exactly one `/`? What happens if there are multiple? Might want to specify `.split('/', 1)`?